tests/integration/extension/src/background.ts

```diff
@@ -2,7 +2,7 @@ import { Rally, RunStates } from "@mozilla/rally-sdk";
 
 const enableDevMode = false;
 const rallySite = "http://localhost:5000";
-const studyId = "facebookPixelHunt";
```

Comment:
It'd be nice to simulate loading multiple extensions at some point, we might want to generate them programmatically though (Firefox does this in tests and I think there's some code in the rally-core-addon repo that does it as well), and using random/simulated data. However, for now this is fine :)